internal/txapp/txapp.go

```diff
@@ -71,6 +77,9 @@ type TxApp struct {
 	// The various internal stores (accounts, votes, etc.) are accessed through
 	// the Database via the functions defined in relevant packages.
 
+	chainParams *chain.GenesisConfig // NOTE: .ConsensusParams can change (be out of date)
```

Comment:
It seems this field is totally unused, aside from when you set it.

Reply:
Yeah, good catch.  I'm pretty certain the note is a message to myself to remove the field on account of the problematic mutability.